bin/bundle

```diff
@@ -1,3 +1,105 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
```

Comment:
Do you know why we need a binstub for Bundler?

Reply:
something something bug in bundler todo with binstubs .... /cc @camelpunch

Reply:
I don't know why bundler (or Rails) introduced a binstub for bundler itself. I assume it's to lock the version of bundler for the app.

The bug that @chrisfarms is referring to is here, but not sure if this is related to the question: https://github.com/bundler/bundler/issues/6149

Reply:
I'm inclined to delete it rather than worry about when we can update/remove the backported fix. It doesn't seem as important the `rails` binstub, which appears to be a standard for Rails apps.

Reply:
The fix was primarily for stubs like `rspec`, which are used a lot. I think the only risk here is if someone:

a) Decides to run e.g. `bundle binstubs rspec-core` and
b) Commits the change

Otherwise if we're concerned about bundler versions, we should downgrade to 1.15, as [one of the comments on the Rails bug thread suggested](https://github.com/rails/rails/issues/31193#issuecomment-346116270), because 1.15 doesn't generate incompatible binstubs by default (1.16 is fine [if you also supply `--force`](https://github.com/rails/rails/issues/31193#issuecomment-347784561))